patches/sirv@2.0.4.patch

```diff
@@ -70,4 +70,4 @@ index c05040fc6ec504a1828a7badd39f669981acd0ee..e9597e8b5bf24613a09565f0e13024ae
 +		shouldServe?: (absoluteFilePath: string) => void;
  	}
  
- 	export default function(dir?: string, opts?: Options): RequestHandler;
\ No newline at end of file
+ 	export default function(dir?: string, opts?: Options): RequestHandler;
```

Comment:
This file actually didn't have any change, except this new line, which latest `pnpm patch` seems to start adding.